arns/arns-token-1.md

```diff
@@ -261,6 +272,142 @@ Send({
 }
 ```
 
+#### Set-Description
+
+Updates the `Description` of this ANT.
+
+Executable by the process `Owner` or an authorized user in the `Controllers` table.
+
+##### Parameters
+
+| Name   | Type   | Description                 |
+| ------ | ------ | --------------------------- |
+| Description | string | The new Description for the ANT. |
+
+##### Rules
+
+- Must be an authorized process `Owner` or `Controller`.
+- Must specify a valid `Description` parameter (string) as a message tag.
+- `Description` must not be longer than 512 characters.
+- Should add `X-`forwarded tags to the response notice.
+
+##### Action
+
+```
+Send({
+  Target = "{Process Identifier}",
+  Action = "Set-Description",
+  Description = "ArDrive is an app that makes it easy to upload, download and share your public or private files on Arweave"
+})
+```
+
+##### Responses
+
+**Permission error, not authorized**
+
+```
+{
+  Target = msg.From,
+  Action = "Invalid-Set-Description-Notice",
+  Data = permissionErr,
+  Error = "Set-Description-Error",
+  ["Message-Id"] = msg.Id
```

Comment:
consider: https://github.com/permaweb/aos/blob/4c545e025c7bbf77cc57e55ba837e13851a90fd6/process/process.lua#L317

Reply:
Not sure if we need to be prescriptive about HOW the message is sent back.  Until we are using msg.reply everywhere, and understand its benefits, lets hold off on mentioning it in the spec itself.  Can come back around and potentially add it once we implement it.